docs/development/triage.rst

```diff
@@ -0,0 +1,229 @@
+
+This refers to Issues only, and it is added when the maintainer(s) cannot
+reproduce the behaviour reported.
+
+.. _triage-deprecation-label:
+
+``deprecation``
+---------------
+
+Added only to PRs, when a API deprecation is introduced.
+
+.. _triage-documentation-label:
+
+``documentation``
+-----------------
+
+Can be added to both Issues and PRs, anything related to the documentation.
+
+.. _triage-duplicate-label:
+
+``duplicate``
+-------------
+
+Applicable to Issues only, to be added before closing a duplicate.
+
+.. _triage-epic-label:
+
+``epic``
+--------
+
+When the Issue/PR is part of a bigger change that involves several other Issues
+and/or PRs (the others must be referenced in the body, or using the ZenHub
+feature).
+
+.. _triage-feature-label:
+
+``feature``
+-----------
+
+When a new feature is introduced, or the user requests a new feature.
+
+.. _triage-good-first-issue:
+
+``good first issue``
+--------------------
+
+While we want to encourage the community to contribute more and more frequent,
+many engineers are still afraid of complex tasks. This label marks simple fixes
+that new contributors can address. It is recommended that this label to be
+accompanied by an explanation and a pointer for the new contributors. 
+
+.. _triage-help-wanted:
+
+``help wanted``
+---------------
+
+This marks an Issue were we ask the community for help, or we need more details
+on a particular topic (e.g., outputs from different platforms, explanation, etc.)
+from any volunteer from the community.
+
+Once we have all the details required, the maintainer has to remove this label
+even though it does not start working on it immediately.
+
+.. _triage-high-severity-label:
+
+``high severity``
+-----------------
+
+Whenever a :ref:`triage-bug-label` affects severely one or more features, making
+it basically unusable.
+
+.. _triage-info-needed-label:
+
+``info needed``
+---------------
+
+We add this label when we need more details and further explanation from the user
+that reports an Issue. Once we received everything needed, we can remove that
+label.
+
+.. _triage-investigation-label:
+
+``investigation``
+-----------------
+
+We need to investigate the problem further.
+
+.. _triage-new-driver:
+
+``new driver``
+--------------
+
+When we discuss the possibility to add a new core driver.
+
+.. _triage-new-method:
+
+``new method``
+--------------
+
+When we discuss the possibility or implement a new method to one or more drivers.
+The method does not necessarily need to be a completely new one to NAPALM.
+
+.. _triage-vendor-bug-label:
+
+``vendor bug``
+--------------
+
+When the bug is casued by a vendor stupidity.
+
+.. _triage-milestone:
+
+Milestone
++++++++++
+
+The milestones are used to group the Issues and the Pull Requests from a
+different angle:
+
+.. _triage-version-milestone:
+
+Version
+-------
+
+The Issue will be solved, or the PR will be included in this release.
+
+.. _triage-approved-milestone:
+
+``APPROVED``
```

Comment:
We should move this to github projects instead.

Reply:
Milestones are a GitHub feature, not ZenHub, and it helps to schedule a issue/PR to a certain release version.